lib/reactotron-core-ui/src/components/TimelineCommand/index.tsx

```diff
@@ -80,6 +80,7 @@ const ChildrenContainer = styled.div`
   animation: fade-up 0.25s;
   will-change: transform opacity;
   padding: 0 40px 30px 40px;
+  word-break: break-word;
```

Comment:
@opakholis thanks for jumping in to help fix this. `break-word` is deprecated so let's use the same `word-break` strategy as we use in the `PreviewContainer` on line 61 here:
```suggestion
  word-break: break-all;
```

Reply:
@morganick thanks for the detailed explanation and suggestion. I didn't realize that `break-word` was deprecated.

Reply:
@opakholis I didn't either until I looked it up to make sure that's the one we wanted. 😅